### PR TITLE
Handle setting donor status by visiting a page

### LINKED ISSUE #377: Segmentation: Donor Status Based On Page View
Add a segmentation option where visits to a specific page segments the user as a donor. The obvious example of this is a donation "thanks" page. Even if the donation occurs offsite, a visit to the thanks page serves as an imperfect yet fairly reliable way to determine donor status in real time.

### PR DESCRIPTION
**Pull request by adekbadek**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #377 .

### How to test the changes in this Pull Request:

_Follow testing steps in https://github.com/Automattic/newspack-plugin/pull/829_

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-newspack-popups-settings.php [hunk 1]
```diff
@@ -110,6 +129,21 @@ public static function get_settings() {
...
 			],
+			[
+				'key'            => 'newspack_popups_donor_landing_page',
+				'value'          => self::donor_landing_page(),
+				'label'          => __(
+					'Donor landing page',
+					'newspack-popups'
+				),
+				'help'           => __(
+					'When using an off-site donation platform, you can set it to landing to a page on your site. When a reader is landinged to this page, they will be marked as a donor.',
```
Comment (dkoo) [inline includes/class-newspack-popups-settings.php:140]:
```suggestion
					'Set a page on your site as a donation landing page. Once a reader views this page, they will be considered a donor. This is helpful if you're using an off-site donation platform but still want to target donors as an audience segment.',
```

**Review by dkoo — APPROVED** [review]
Looks good and works as described. Just a small suggestion on the help text for the new Settings option.

**Review by dkoo — CHANGES_REQUESTED** [review]
Whoops, approved by accident. Requesting changes.

File: includes/class-newspack-popups-settings.php (hunk 1 above)
Comment (adekbadek) [reply includes/class-newspack-popups-settings.php:140]: "landinged" 😂  🤦‍♂️

**Review by dkoo — APPROVED** [review]

**Pull request merged**